addons/mail/static/src/core/common/message_service.js

```diff
@@ -50,6 +50,7 @@ export class MessageService {
         this.store.Message.insert(
             Object.assign(messageData, {
                 body: messageData.body ? markup(messageData.body) : messageData.body,
+                translationValue: false,
```

Comment:
This value should be received from the RPC return + bus notification instead so that multiple clients/tabs can discard the value accordingly.